.github/workflows/ci.yml

```diff
@@ -71,6 +71,33 @@ jobs:
           cat bench.log
           export TEST_CMD=`grep Running fuse_cargo_test.log | awk '{print $2}'`
           sudo $TEST_CMD
+  csi-cargo-test:
+    name: CSI-Cargo-Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install CSI dependencies
+        run: sudo apt install -y cmake g++ libprotobuf-dev protobuf-compiler
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Prepare Rust environment
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: default
+          toolchain: ${{ env.CI_RUST_TOOLCHAIN }}
+          override: true
+      - name: Build bind_mounter
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --bin bind_mounter
+      - name: Enable bind_mounter permission and run cargo test
+        run: |
+          sudo chown root:root $BIND_MOUNTER
+          sudo chmod u+s $BIND_MOUNTER
+          ls -lsh $BIND_MOUNTER
+          export ETCD_END_POINT=http://127.0.0.1:2379
+          export BIND_MOUNTER=`realpath $BIND_MOUNTER`
+          cargo test -p csi
```

Comment:
also run cargo test with root to test direct mount, see
https://github.com/datenlord/datenlord/blob/f1240e3ffc3cd6516b034749d3e8216bbbc84465/.github/workflows/ci.yml#L72